server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/ChangeParentAction.java

```diff
@@ -19,38 +19,30 @@
  */
 package org.sonar.server.qualityprofile.ws;
 
-import com.google.common.base.Preconditions;
+import org.elasticsearch.common.Strings;
```

Comment:
you can exclude `org.elasticsearch.common.Strings` in IntelliJ because I suppose you wanted to use the Guava version